### PR TITLE
Update American Social Security Number.

### PR DESCRIPTION
**Pull request by RackReaver**
- No numbers with all zeros in any digit group
- No numbers with 666 or 900–999 (Individual Taxpayer Identification Number) in the first digit group.
- Account for potential digit group spacers `= | \+ | _ | \# | : | ; | \. | - | \s` 

References:
https://en.wikipedia.org/wiki/Social_Security_number
https://www.usrecordsearch.com/ssn.htm

### CODE REVIEW

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -463,6 +463,19 @@
           "Credentials"
        ]
     },
+    {
+      "Name": "American Social Security Number",
+      "Regex": "([0-9][0-9][0-9])(?<!666|000|9[0-9][0-9])(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}([1-9][1-9]|0[1-9]|[1-9]0)(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}([0-9][0-9][0-9][0-9])(?<!0000)(=|\\+|_|\\#|:|;|\\.|-|\\s){0,3}?",
+      "plural_name": false,
+      "Description": null,
```
Comment (piatrashkakanstantinass) [inline pywhat/Data/regex.json:470]: We had description before. Can you use it here?

**Review by piatrashkakanstantinass — CHANGES_REQUESTED** [review]

File: pywhat/Data/regex.json (hunk 1 above)
Comment (RackReaver) [reply pywhat/Data/regex.json:470]: I didn't understand what use it had so I removed it. Would you like me to put it back?

Comment (piatrashkakanstantinass) [reply pywhat/Data/regex.json:470]: Yes

Comment (RackReaver) [reply pywhat/Data/regex.json]: Done

**Comment by bee-san** [comment]
Ah sorry!!! I tried to merge with master and deleted your changes :-( Can you add them back from here, please? https://github.com/bee-san/pyWhat/pull/95/commits/fa5a5129099f18ee23a75d78b439c6e93cf2e6b2

**Pull request closed without merging**